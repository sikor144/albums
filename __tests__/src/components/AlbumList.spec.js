import React from 'react';
import renderer from 'react-test-renderer';

import AlbumList from '../../../src/components/AlbumList';

describe('AlbumList component', () => {
  it('renders component correctly', () => {
    const component = renderer.create(
      <AlbumList />,
    ).toJSON();

    expect(component).toMatchSnapshot();
  });
});
