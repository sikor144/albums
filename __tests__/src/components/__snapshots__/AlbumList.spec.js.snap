// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AlbumList component renders component correctly 1`] = `
<RCTScrollView>
  <View />
</RCTScrollView>
`;
